_drafts/technical-interviews.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout:       post
+title:        Nailing the Technical Interview
```

Comment:
Putting SEO hat on....

"Nailing the Technical Interview" --> "How To Nail the Technical Interview". Take it with a grain of salt either way. Maybe "How To Pass?" idk... thesaurus TIME!